corehq/motech/dhis2/models.py

```diff
@@ -471,3 +514,36 @@ def get_ucr_data(report_config, date_filter, date_span):
     filter_values = get_filter_values(report_config.ui_filters, filter_params)
     data_source.set_filter_values(filter_values)
     return data_source.get_data()
+
+
+def get_end_of_period(frequency, period):
```

Comment:
This function is pining for tests

Reply:
Yes! Your're right! I only tested through console so forgot to actually include the test.